Packs/GSuiteAdmin/Integrations/GSuiteAdmin/README.md

```diff
@@ -1870,4 +1870,619 @@ There is no context output for this command.
+| policy_field_json_entry_id | Entry Id of the file that contains JSON representation of the ModifyGroupPolicyRequest. | Optional | 
+| target_resource | The target resource on which this policy is applied. The following resources are supported:<br/><br/>Organizational Unit ("orgunits/{orgunit_id}")<br/>Group ("groups/{groupId}"). | Optional | 
+| additional_target_keys | Map containing the additional target key name and value pairs used to further identify the target of the policy.<br/><br/>A string containing a list of "key": "value" pairs. Example: ""name": "wrench", "mass": "1.3kg", "count": "3"". | Optional | 
+| policy_schema | The fully qualified name of the policy schema that is being inherited. | Optional | 
+| policy_value | The value of the policy that is compatible with the schema that it is associated with Struct format. | Optional | 
+| update_mask | Policy fields to update. Only fields in this mask will be updated; other fields in policyValue will be ignored (even if they have values). If a field is in this list it must have a value in 'policyValue'.<br/>This is a comma-separated list of fully qualified names of fields. Example: "user.displayName,photo".<br/>Field mask format. | Optional | 
+
+#### Context Output
+
+There is no context output for this command.
+
+#### Command Example
+```!gsuite-policy-modify customer_id=test target_type=Group target_resource=1111111111 policy_schema_filter=chrome.users.apps.InstallType additional_target_keys="\"app_id\":\"chrome:1111111aaaaaaaaa\"" policy_schema=chrome.users.apps.InstallType policy_value=BLOCKED update_mask=appInstallType```
+
+#### Human Readable Output
+> Policy has been modified for the customer CUSTOMER_ID.
+
+### gsuite-policy-schemas-list
+
+***
+Gets a list of policy schemas that match a specified filter value for a given customer.
+
+#### Base Command
+
+`gsuite-policy-schemas-list`
+
+##### Required Permissions
+`https://www.googleapis.com/auth/chrome.management.policy`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| customer_id | ID of the Google Workspace account or literal "my_customer" for the customer associated with the request. | Required | 
+| schema_name | Required when asking for a specific policy. The policy schema resource name to query. | Optional | 
+| filter | The schema filter is used to find a particular schema based on fields like its resource name, description and additionalTargetKeyNames. | Optional | 
+| page_size | The maximum number of policy schemas to return, defaults to 100 and has a maximum of 1000. Default is 100. | Optional | 
+| page_token | Page Token. | Optional | 
+| limit | Limit. | Optional | 
```

Comment:
Limit of what? The number of devices per retrieval? The number of devices per page?